app/views/layout.njk

```diff
@@ -41,7 +37,7 @@
         </div>
         <div class="govuk-header__content">
           <span class="govuk-header__link--service-name">
-            {{serviceName}}
+            {{productName}}
```

Comment:
The new pages should use a separate layout template that displays the product name rather than the service name - we want to keep it as-is for the current payment link journey until switching over I think.

Reply:
Good catch - just created a separate layout for new pages.